src/components/editMeasure/populationCriteria/QDMReporting/QDMReportingValidator.ts

```diff
@@ -0,0 +1,12 @@
+import * as Yup from "yup";
+
+export const QDMReportingValidator = Yup.object().shape({
+  improvementNotation: Yup.string(),
```

Comment:
We are not doing any validation here, do we need this ?

Reply:
This is preventing the Save button from being active if Other is selected in the first drop down and the text box isn't filled in